charmcraft/parts/plugins/_poetry.py

```diff
@@ -25,8 +25,8 @@
 
 
 class PoetryPluginProperties(poetry_plugin.PoetryPluginProperties, frozen=True):
-    poetry_keep_bins: bool = False
-    """Keep the virtual environment's 'bin' directory."""
+    poetry_keep_bin_activate: bool = True
```

Comment:
Let's still leave an option to keep the whole `bins` directory, in case people want it for some reason.